request.go

```diff
@@ -166,10 +166,10 @@ type ConvStoreDetail struct {
 	Message string `json:"message"`
 }
 
-// GopayDetail: Represent gopay detail
+// GopayDetail : Represent gopay detail
 type GopayDetail struct {
 	EnableCallback bool   `json:"enable_callback"`
-	CallbackUrl    string `json:"callback_url"`
+	CallbackURL    string `json:"callback_url"`
```

Comment:
Let's keep `CallbackUrl` to avoid backward compatibility issue to merchant who already use it.